src/oc_erchef/apps/oc_chef_wm/src/chef_wm_rabbitmq_management.erl

```diff
@@ -302,3 +241,14 @@ parse_integer(Val) when is_list(Val) ->
         {Int, _Rest} -> Int
     end;
 parse_integer(_) -> undefined.
+
+-spec check_aliveness(atom(), string()) -> true | false.
```

Comment:
[nit] `true | false` can be boolean().  You can find the various built-in typespecs here: http://erlang.org/doc/reference_manual/typespec.html